tools/devservices_healthcheck.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import subprocess
+from time import sleep
+from typing import List
+
+
+def run_cmd(
+    args: List[str],
+    *,
+    retries: int = 3,
+    timeout: int = 5,
+) -> int:
+    cmd = " ".join(args)
+    for retry in range(1, retries + 1):
+        logging.info(f"Running {retry} health check {cmd!r}...")
+        try:
+            returncode = subprocess.call(args)
+        except Exception:
+            continue
```

Comment:
`subprocess.call` returns the exit code and won't raise an `Exception` -- the only case this raises is if the executable is not found (`OSError` / `FileNotFoundError`) which _should_ bubble up as it's a programming error

general advice as well: always catch a more specific exception than `Exception` / `BaseException`